### PR TITLE
Library kind, admin stub

### PR DESCRIPTION
**Pull request by andrewconner**
@ahsu1230

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminLibraryController.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  def index() = AdminHtmlAction.authenticated { implicit request =>
+    Ok
+  }
+
+  def internUserSystemLibraries(userId: Id[User]) = AdminHtmlAction.authenticated { implicit request =>
+    val res = libraryCommander.internSystemGeneratedLibraries(userId)
+
+    Ok(res.toString)
+  }
+}
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminLibraryController.scala:25]: What exactly is this file for again? an endpoint for the admin kifi site?

File: kifi-backend/modules/common/app/com/keepit/model/Library.scala [hunk 2]
```diff
@@ -96,3 +97,24 @@ object LibraryVisibility {
...
+    Format(__.read[String].map(LibraryKind(_)), new Writes[LibraryKind] { def writes(o: LibraryKind) = JsString(o.value) })
+
+  def apply(str: String) = {
+    str match {
+      case SYSTEM_MAIN.value => SYSTEM_MAIN
+      case SYSTEM_SECRET.value => SYSTEM_SECRET
+      case USER_CREATED.value => USER_CREATED
+    }
+  }
+}
```
Comment (ahsu1230) [inline kifi-backend/modules/common/app/com/keepit/model/Library.scala:120]: this other style is working out pretty well =)

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminLibraryController.scala (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminLibraryController.scala:25]: Yep. I'll use it to make libraries for everyone. Don't have the commander methods needed yet though.

**Comment by ahsu1230** [comment]
lgtm! I'll have my library push soon as well once I merge these files

**Pull request merged**